### ISSUE TITLE
Private Stub

### ISSUE THREAD

**Issue by JackTheTripperr** [bytecode77/pe-union#33]
Hi Byte! Love all your work, some really cool stuff you've discovered and built. I've been learning about crypters as of late and have managed to cobble together a frankensteined version of a couple different crypter projects and what do you know...it actually works! While I did learn a lot about process injection and cryptology, the sad truth is that I know my way around LLM's far better than I can actually write code. I have no problem admitting that...if it works, it works, right?

Anyhow, in another open issue on this repository I saw that you had offered to point someone in the direction of some resources that might help them learn to "privatize" the public version of the stub that comes standard with this project. I'd love to pick up enough knowledge to be able to do such a thing, and I was wondering if you'd be willing to point me towards some resources that might get me there?

TLDR: Where can I learn about "privatizing" the stub that comes with public projects like this one?

Thanks!

**Comment by bytecode77** [comment]
That's the number one question, of course. I figure that you already know where the stub code files are that need modification and you're looking for ways to FUD a stub in general.

The most important tool is AntiScan.me or similar, which unlike VirusTotal do not share your submitted files with AV vendors. You can use these tools for trial & error, which you will need to do a lot.

Also I watch a lot of DEF CON presentations around the topic. [Here](https://www.youtube.com/watch?v=S8kCCeCEALE&list=PLLT6XEaj7l-EsQhhxPSvKOo_xKkt68ilb&index=1)'s a playlist that I just found, or [this](https://www.youtube.com/watch?v=edIMUcxCueA&t=8s) or [this](https://www.youtube.com/watch?v=w0bh7s7bVXI&t=6s). But there are a lot more good content on evasion, which I just cant find right now...

The most important distinction you need to make is scan-time evasion, anti emulation, and EDR evasion.
Good luck ;)

**Comment by JackTheTripperr** [comment]
> you're looking for ways to FUD a stub in general

That's exactly it! I'm in my infancy when it comes to "evasion" in general, and it's really hard not to get sidetracked only to find yourself crawling out of a very deep rabbit hole a couple of days later. That's the fun of it though, isn't it? Occasionally I'll get something to work, and then immediately proceed to panic over whether or not my next action is going to kill my shell. Good times!

I really appreciate your reply and look forward to digesting the materials you've suggested. Thanks!
